### ISSUE TITLE
Celeste added new "emblems" without user permission, cannot remove

### ISSUE THREAD

**Issue by AchillesBoi** [hwittenborn/celeste#148]
I installed Celeste through Flatpak and now I get ugly, empty icons on the Emblem's section of Nemo (file manager) that are also stretching the Properties window much wider than the default size. Please help me remove this.

How it looks like:
![image](https://github.com/hwittenborn/celeste/assets/55808665/9ab92597-9d65-440a-a330-eeaacf039c4f)

How it should look like:
![image](https://github.com/hwittenborn/celeste/assets/55808665/c8fbb55f-fa6f-4a3c-8004-78b48a280c9a)

**Comment by hwittenborn** [comment]
Hey! I'm not really familiar with Nemo - what are the purpose of emblems? Are they just supposed to be used as the icon of a file?

**Comment by hwittenborn** [comment]
I'll say too that GNOME is the primary desktop environment (and in turn using with Nautilus) that Celeste aims to support. I'm still open to trying to get this fixed up, but if you want the best experience I'd definitely recommend using GNOME.

**Comment by AchillesBoi** [comment]
> Hey! I'm not really familiar with Nemo - what are the purpose of emblems? Are they just supposed to be used as the icon of a file?

Nemo is a fork of Nautilus from back when it had a lot more features like split-view and icon emblems.

> I'll say too that GNOME is the primary desktop environment (and in turn using with Nautilus) that Celeste aims to support. I'm still open to trying to get this fixed up, but if you want the best experience I'd definitely recommend using GNOME.

I am using GNOME 42.x at the moment (latest version from Ubuntu 22.04.3) but using Nemo as my file manager. It seems that after installing Celeste, these new icon-less emblems started appearing in the Emblems section of the Properties window in Nemo.

I've uninstalled Celeste and they're still there. It's super weird too because I installed the Flatpak version which as I understand it should be in its own container and away from system stuff.

**Comment by hwittenborn** [comment]
> Nemo is a fork of Nautilus from back when it had a lot more features like split-view and icon emblems.

> I am using GNOME 42.x at the moment (latest version from Ubuntu 22.04.3) but using Nemo as my file manager. It seems that after installing Celeste, these new icon-less emblems started appearing in the Emblems section of the Properties window in Nemo.

I wasn't aware of that, thanks for letting me know that all.

> I've uninstalled Celeste and they're still there. It's super weird too because I installed the Flatpak version which as I understand it should be in its own container and away from system stuff.

Had you installed Celeste from a Deb/Snap before, or have you only used the Flatpak? That's definitely a bit weird if it's popping up from only using the Flatpak, though I'm not doubting it happened somehow.

I did some local searching of those emblems that popped up and it looks like they might be from the Yaru icon theme - do you have that installed on your host system? I'm pretty sure Celeste has a copy of the Yaru icon theme bundled in the Flatpak and I'm suspecting that might be the cause if you didn't already have it installed with APT.

**Comment by AchillesBoi** [comment]
> Had you installed Celeste from a Deb/Snap before, or have you only used the Flatpak? That's definitely a bit weird if it's popping up from only using the Flatpak, though I'm not doubting it happened somehow.

Yep, only installed the Flatpak version, first time trying this app.

So a bit more information:
I'm currently using Linux Mint 21.2 (latest version) which is based on Ubuntu 22.04.3 and, while it usually comes with Cinnamon DE, I installed GNOME 42.x by installing gnome-session which installs everything needed for GNOME. It runs like a dream, and I've been using GNOME with Mint for a few years now without a hiccup. Think of it as Ubuntu but with a good file manager and Flatpak-centric instead of Snap.

![Screenshot from 2023-10-06 23-49-43](https://github.com/hwittenborn/celeste/assets/55808665/7084adf6-7d9a-45ae-80af-bfeb7b78c70b)

> I did some local searching of those emblems that popped up and it looks like they might be from the Yaru icon theme - do you have that installed on your host system? I'm pretty sure Celeste has a copy of the Yaru icon theme bundled in the Flatpak and I'm suspecting that might be the cause if you didn't already have it installed with APT.

The latest version of Mint uses Mint-Y icons by default, but Yaru and Adwaita icons are installed by default as well. Can you tell me a bit of what the Flatpak version of Celeste does to add emblem icons to Nautilus? Since Nemo is a fork of Nautilus, most of the original Nautilus configuration applies to Nemo as well, so that's why it's being affected by it.

**Comment by AchillesBoi** [comment]
Were you able to find the problem? If not, can you at least tell me what Celeste does to install these icon-less emblems on my machine? That way so I can manually delete them or something.
